src/inventory/ModelManager.h

```diff
@@ -78,24 +85,24 @@ struct Model {
             } else if (key == "remoteAPI") {
                 remoteAPI = val;
             } else {
-                parseError = true;
+                parseError = true; // TODO Idem.
             }
         } else if constexpr (std::is_same_v<QJsonObject, T>) {
             if (key == "srcTags") {
                 srcTags = val.toVariantMap();
             } else {
-               parseError = true;
+               parseError = true; // TODO Again
             }
         }
         if (parseError) {
-            std::cerr << "Unknown key type. " << key.toStdString() << " Something is very wrong!" << std::endl;
+            qDebug() << "Model::set() was called with an unknown key type for key" << key;
```

Comment:
we shouldn't be spitting on the command line, right? Although this is probably a developer error, if we end here?

Reply:
Yeah, developer error but again one that doesn't do any harm. We have `validateModel()` already for checking whether a model has all necessary data. This one is only about calling `set()` with the wrong type, or with an unknown `key`.

And again, we can easily hide this behind a `--debug` flag